rails-tasks/config/routes.rb

```diff
@@ -1,8 +1,14 @@
 Rails.application.routes.draw do
   root 'tasks#index'
-  get '/task_list' => 'tasks#index'
-  get 'task_list/new' => 'tasks#new'
-  get 'task_list/:name' => 'tasks#show'
+
+  get 'tasks' => 'tasks#index'
```

Comment:
This change was done before we talked about it in class, I think, but this shift from `task_list` to `tasks` is a perfect example of why the `link_to` method, along with the "route info hash" (e.g. `{controller: "tasks", action: "index"}`) is preferable.
In that case even if you change your routes to use `tasks` instead of `task_list` all of your links will work automagically!